pyscript.conf

```diff
@@ -1,24 +1,25 @@
 [homeassistant]
-#
-# Set hass_host to the host name or IP address where your HASS is running.
-# This should be a host name or IP address you can ping from the Jupyter
-# client machine. e.g., 192.168.1.10 or homeassistant.
-#
-# Do not use quotes
-#
-hass_host = YOUR_HASS_HOST_OR_IP
-#
-# Set hass_url to the URL of your HASS http interface. Typically the host
-# name portion will be the same as hass_host above.
-# e.g., http://my-homeassistant.duckdns.org/ or http://192.168.1.10:8123.
-#
-# Do not use quotes
-#
-hass_url = http://YOUR_HASS_HOST_OR_IP:8123
-#
-# Set hass_token to a long-term access token created via the button
-# at the bottom of your user profile page in HASS.
-#
-# Do not use quotes
-#
-hass_token = REPLACE_WITH_THE_LONG_TERM_ACCESS_KEY_FROM_HASS
+
+## Set hass_host to the host name or IP address where your HASS is running.
+## This should be a host name or IP address you can ping from the Jupyter
+## client machine. e.g., 192.168.1.10 or homeassistant.
+## default: localhost
+# hass_host = localhost
```

Comment:
While I do see you added defaults, I'd like for the three required parameter settings to not be commented.  Also, the comments should have single "#", not double.  It's ok for the hass_proxy setting to remain commented.